src/libraries/System.Formats.Nrbf/src/System.Formats.Nrbf.csproj

```diff
@@ -4,9 +4,6 @@
     <TargetFrameworks>$(NetCoreAppCurrent);$(NetCoreAppPrevious);$(NetCoreAppMinimum);netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
     <EnableDefaultItems>true</EnableDefaultItems>
     <CLSCompliant>false</CLSCompliant>
```

Comment:
Adding a few suggestions.  Also consider adding a Package.md with more information about the package, using the same template as other projects in the repo.

```suggestion
    <CLSCompliant>false</CLSCompliant>
    <UseCompilerGeneratedDocXmlFile>true</UseCompilerGeneratedDocXmlFile
    <IsPackable>true</IsPackable>
    <PackageDescription>Provides a safe reader for BinaryFormatted payloads.

Commonly Used Types:
System.Formats.Nrbf.NrbfDecoder</PackageDescription>    

    <!-- Disabling baseline validation since this is a brand new package.
         Once this package has shipped a stable version, the following line
         should be removed in order to re-enable validation. -->
    <DisablePackageBaselineValidation>true</DisablePackageBaselineValidation>
```